### PR TITLE
fix(notification): #CRRE-506 fix admins notification, changed from or…

### PR DESCRIPTION
**Pull request by Sofianebernoussi**
## Describe your changes

- Fix count system (from orders to projects count) following #60

## Checklist tests

- Check if the count of new order is correct.
- Check if the count take only new orders.

## Issue ticket number and link
[CRRE-506](https://jira.support-ent.fr/browse/CRRE-506)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/crre/service/impl/DefaultNotificationService.java [hunk 1]
```diff
@@ -52,16 +57,46 @@ private boolean isProjectIsComplete(Map.Entry<ProjectModel, List<OrderRegionEqui
...
-                .onSuccess(compositeResult -> {
-                    JsonObject newOrderResult = newOrderCountFuture.result();
-                    if (adminUserFuture.result().size() > 0 && !newOrderResult.isEmpty() &&
-                            newOrderResult.getInteger(Field.NB_ORDER.toLowerCase()) > 0) {
-                        adminUserFuture.result()
-                                .forEach(adminUser -> this.prepareMessageToAdmin(adminUser.getUserId(), newOrderResult.getInteger(Field.NB_ORDER.toLowerCase())));
+        List<Integer> projectIdList = new ArrayList<>();
+        Map<ProjectModel, List<OrderRegionEquipmentModel>> projectModelListMapAll = new HashMap<>();
+        Map<ProjectModel, List<OrderRegionEquipmentModel>> projectModelListMap = new HashMap<>();
+        this.serviceFactory.getProjectService().getAllProject()
```
Comment (sjollois) [inline src/main/java/fr/openent/crre/service/impl/DefaultNotificationService.java:63]: Tu ne pourrais pas filtrer ? car en Prod il y a plus de 1000 project

Comment (sjollois) [reply src/main/java/fr/openent/crre/service/impl/DefaultNotificationService.java:63]: les perfs risquent d'en pâtir non ?

File: src/main/java/fr/openent/crre/service/impl/DefaultProjectService.java [hunk 2]
```diff
@@ -14,6 +14,18 @@
 
 public class DefaultProjectService implements ProjectService {
 
+    public Future<List<ProjectModel>> getAllProject() {
```
Comment (BastienBenard) [inline src/main/java/fr/openent/crre/service/impl/DefaultProjectService.java:17]: @Override

**Review by qpic-cgi — APPROVED** [review]

File: src/main/java/fr/openent/crre/service/impl/DefaultNotificationService.java (hunk 1 above)
Comment (Sofianebernoussi) [reply src/main/java/fr/openent/crre/service/impl/DefaultNotificationService.java:63]: Filtrer par ? j'ai besoin de tout

**Review by BastienBenard — APPROVED** [review]

**Pull request merged**